manual/wixtoolset/tools/chocolateyInstall.ps1

```diff
@@ -1,7 +1,13 @@
-﻿$packageName = 'wixtoolset'
-$installerType = 'EXE'
-$32BitUrl = 'http://wixtoolset.org/downloads/v3.10.3.3007/wix310.exe'
-$silentArgs = '/q'
-$validExitCodes = @(0)
+﻿$packageParameters = @{
+  PackageName = 'wixtoolset'
+  SilentArgs = '/q'
+  Url = 'http://wixtoolset.org/downloads/v3.11.0.1507/wix311.exe'
+  Url64bit = ''
+  ValidExitCodes = @('0')
```

Comment:
This seems like many lines changed that don't have to be. if you rebase from #643 you should just have to edit lines 4, and 6 (line 6 of #643 )